modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc

```diff
@@ -201,7 +201,11 @@ To avoid this error, set a limit range for the namespace, where the defaults fro
 
 For more information about setting limit ranges, refer to _Restrict resource consumption with limit ranges_ in the _Additional resources_ section.
 ====
-
++
+[NOTE]
+====
+Since {pipelines-shortname} 1.17, the priority class that you set for a task applies to all pods created for the task, including the affinity assistant pod that {pipelines-shortname} creates in order to ensure that the task executes on a particular node.
```

Comment:
```suggestion
Since {pipelines-shortname} 1.17, support for priorityClassName in in affinityAssistancePodTemplate will be added which will help to specify default priorityClassName in affinity assistant podTemplate for affinity assistant pods. Also value specified in pipelinerun/taskrun will be used for both affinity assistant pods and taskrun pods.
```